docs/messaging/messaging-spans.md

```diff
@@ -184,18 +184,19 @@ If the destination name is dynamic, such as a [conversation ID](#conversations)
 In these cases, an artificial destination name that best expresses the destination, or a generic, static fallback like `"(anonymous)"` for [anonymous destinations](#temporary-and-anonymous-destinations) SHOULD be used instead.
 
 The values allowed for `<operation name>` are defined in the section [Operation names](#operation-names) below.
-If the format above is used, the operation name MUST match the `messaging.operation` attribute defined for message consumer spans below.
 
 Examples:
 
 * `shop.orders publish`
 * `shop.orders receive`
-* `shop.orders process`
+* `shop.orders settle`
 * `print_jobs publish`
-* `topic with spaces process`
-* `AuthenticationRequest-Conversations process`
+* `topic with spaces deliver`
+* `AuthenticationRequest-Conversations settle`
 * `(anonymous) publish` (`(anonymous)` being a stable identifier for an unnamed destination)
 
+Messaging system specific adaptions to span naming MUST be documented in [semantic conventions for specific messaging technologies](#semantic-conventions-for-specific-messaging-technologies).
```

Comment:
Google Pub/Sub plans to differ from this naming slightly. `extend` and `settle` are replaced with `modack` (Modify Ack Deadline) and `ack` (acknowledge).

The concept of settlement is not known within our documentation so we're worried it may confuse users. This will be documented in the section described though.

Reply:
@hongalex so are you good with the way it is structured/mentioned now? Or you think we need to change something?

Reply:
@hongalex Yes, that's exactly what this clause is for. You can use different span and operation names, as long as you document it in the system-specific portions of the conventions.

Reply:
Yeah I just wanted to provide heads up as to why the naming might look different.

The actual language looks good though!